lib/components/admin/trip-status.js

```diff
@@ -47,9 +48,15 @@ class TripStatus extends Component {
     ? <Icon type='check' className='text-success' />
     : <Icon type='exclamation-circle' className='text-warning' />
 
-  _onPlanTrip = () => this.props.planTrip(this.props.request, this.props.outbound)
+  _onPlanTrip = () => {
+    const { outbound, planTrip, request } = this.props
+    planTrip(request, outbound)
+  }
 
-  _onSaveTrip = () => this.props.saveRequestTrip(this.props.request, this.props.outbound)
+  _onSaveTrip = () => {
+    const { outbound, request, saveRequestTripItineraries } = this.props
+    saveRequestTripItineraries(request, outbound)
+  }
```

Comment:
It wasn't super clear in the ticket, but I think we also need a `viewTrip` method/button that will load the an already saved trip into the main panel.

Reply:
Implemented this functionality in the last few commits.